jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryAuthenticator.java

```diff
@@ -249,7 +248,24 @@ public Authorization authenticatePush() throws RegistryAuthenticationFailedExcep
 
   @VisibleForTesting
   URL getAuthenticationUrl(String scope) throws MalformedURLException {
```

Comment:
Now I feel I want to remove this. The name is too general while not being used for the OAuth2 path. We can then have a test for `
getBasicAuthRequestParameters()` instead.

Reply:
see above reply.

Reply:
Will take of this later myself.